fuelup-init.sh

```diff
@@ -118,9 +150,33 @@ main() {
     printf '\n'
     printf '%s\n' "fuelup ${_fuelup_version} has been installed in $FUELUP_DIR/bin. To fetch the latest forc and fuel-core binaries, run 'fuelup install'." 1>&2
 
+    if [ "$allow_modify" = "yes" ]; then
+        if echo "$PATH" | grep -q "$FUELUP_DIR/bin"; then
+            printf "\n%s/bin already exists in your PATH.\n" "$FUELUP_DIR"
+        else
+            echo "export PATH=\"\$HOME/.fuelup/bin:\$PATH"\" >>"$SHELL_PROFILE"
+            printf "\n%s added to PATH. Run 'source %s' or start a new terminal session to use fuelup.\n" "$FUELUP_DIR" "$SHELL_PROFILE"
+        fi
+    fi
+
     return "$_retval"
 }
 
+preinstall_confirmation() {
+    cat 1>&2 <<EOF
+
+fuelup uses $FUELUP_DIR as its home directory to manage the Fuel toolchain, and will install binaries there.
+
+To use the toolchain, you will have to configure your PATH, which tells your machine where to locate fuelup and the Fuel toolchain.
+
+If permitted, fuelup-init will configure your PATH for you:
+
+echo "export PATH="\$HOME/.fuelup/bin:\$PATH"" >> $SHELL_PROFILE
```

Comment:
Will this actually work? Looks like the inner `"`s will break.

Reply:
it works fine running it on `sh` manually for me, is there a reason this would break?